sceptre/plan/plan.py

```diff
@@ -440,3 +440,13 @@ def dump_config(self, *args):
         """
         self.resolve(command=self.dump_config.__name__)
         return self._execute(self.config_reader, *args)
+
```

Comment:
You ought to deprecate `generate` above as well.

Reply:
done